testutil/compose/static/lighthouse/Dockerfile

```diff
@@ -1,10 +1,16 @@
-FROM sigp/lighthouse:latest-amd64-unstable-dev
+FROM sigp/lighthouse:latest-${TARGETARCH}-unstable-dev
```

Comment:
how are we supplying this `TARGETARCH` variable?

Reply:
This is defined by Docker as fas as I get from the documentation.

Reply:
Tested on Mac & on Windows (WSL - Ubuntu).